gpflow/models/model.py

```diff
@@ -118,16 +119,43 @@ def predict_f_samples(
     ) -> tf.Tensor:
         """
         Produce samples from the posterior latent function(s) at the input points.
+
+        :param Xnew: DataPoint
+            Input locations at which to draw samples
+        :param num_samples: int
+            Number of samples to draw.
+            If `None` a one sample is drawn and the return shape is [..., N, P],
+            for any other positive integer the return shape contains an extra batch
+            dimension, [..., S, N, P], with S = num_samples.
+        :param full_cov: bool, default to `True`,
```

Comment:
```suggestion
        :param full_cov:
```
type and default in signature - don't repeat yourself ;)